core/src/main/java/org/frankframework/http/rest/ApiListener.java

```diff
@@ -231,16 +237,29 @@ public boolean accepts(@Nullable String acceptHeader) {
 	}
 
 	/**
-	 * HTTP method to listen to
+	 * HTTP method(s) to listen to, separated by comma
 	 * @ff.default GET
 	 */
-	public void setMethod(HttpMethod method) {
-		this.method = method;
-		if(this.method == HttpMethod.OPTIONS) {
+	public void setMethod(String method) {
+		this.method = StringUtil.splitToStream(method)
+				.map(s -> EnumUtils.parse(HttpMethod.class, s))
+				.collect(Collectors.toList());
+
+		if(hasMethod(HttpMethod.OPTIONS)) {
 			throw new IllegalArgumentException("method OPTIONS should not be added manually");
 		}
 	}
 
+	public String getMethod(){
+		return method.stream()
+				.map(HttpMethod::name)
+				.collect(Collectors.joining(","));
+	}
```

Comment:
> The name of the method suggests it returns only 1 method. I'd suggest something like: `getAllMethods` or `getMethodsAsString`.. or something better.

I looked at that, thought about it, then realised that we most likely need it this way for compatibility with all existing configurations.

Perhaps we can keep the old attribute for backwards compatiblity and name it `methods` internally, deprecating the old name in 8.1.
I'm not sure if any front-end code or any code touching the configuration might break if this method would return the list instead of a string.

Reply:
Thanks @Laurens-makel. Good improvement. I'll resolve this comment.